internal/node/node.bzl

```diff
@@ -200,7 +200,7 @@ def _nodejs_binary_impl(ctx):
     node_tool_files.append(ctx.file._bazel_require_script)
 
     if not ctx.outputs.templated_args_file:
-        templated_args = ctx.attr.templated_args
+        templated_args = [a for a in ctx.attr.templated_args]
```

Comment:
Try using `.to_list()`, I'm not 100% sure it works, but if it does, it's nicer than an identity list comprehension.